homeassistant/components/suez_water/quality_scale.yaml

```diff
@@ -1,9 +1,9 @@
 rules:
   # Bronze
-  config-flow: todo
+  config-flow: done
   test-before-configure: done
   unique-config-entry: done
-  config-flow-test-coverage: todo
+  config-flow-test-coverage: done
```

Comment:
```suggestion
  config-flow-test-coverage: todo
```

Reply:
test are already done.
When I first pushed the quality_scale someone told me not to mark test as done because config flow wasn't completed, even if the only thing missing was the data_description